.github/workflows/release-please-pr-update-tagged-references.yml

```diff
@@ -0,0 +1,142 @@
+# Creates an automated commit whenever a release-please PR is created or updated.
+# The commit updates the references for all actions used in all workflows, actions
+# and examples in the repository to point to the new version tag, which will be
+# created when the PR is merged.
+name: "release-please PR: Update tagged references"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths: .release-please-manifest.json
+
+permissions:
+  contents: read
+
```

Comment:
Should we add a `concurrency` group to both workflows to avoid them running in parallel?